web-server/v0.4/src/components/PageHeader/index.js

```diff
@@ -204,6 +207,29 @@ export default class PageHeader extends PureComponent {
     if (tabActiveKey !== undefined) {
       activeKeyProps.activeKey = tabActiveKey;
     }
+    let titleHeader = '';
+    if (selectedControllers !== undefined) {
+      titleHeader = selectedControllers.map(i => (
+        <Tag color="blue" style={{ fontSize: '105%' }}>
+          {i}
+        </Tag>
+      ));
+      if (selectedControllers.length > 1) {
+        titleHeader = (
+          <Collapse defaultActiveKey={['1']} style={{ marginBottom: '0.5%' }}>
```

Comment:
If we plan on using `Tag` to display controller names, we can simply render the tags without the `Collapse` component as the tags will be wrapped in the `PageHeader` component. We can use this approach for `selectedControllers` with a length of 1 or more.